src/main/java/com/gtnewhorizons/angelica/mixins/TargetedMod.java

```diff
@@ -7,6 +7,7 @@ public enum TargetedMod {
       VANILLA("Minecraft", null)
     , FASTCRAFT("FastCraft", "fastcraft.Tweaker")
     , OPTIFINE("Optifine", "optifine.OptiFineForgeTweaker", "Optifine")
+    , ARCHAICFIX("ArchaicFix", "com.archaicraft.archaicfix.ArchaicFixCore", "archaicfix")
```

Comment:
This should be
```suggestion
    , ARCHAICFIX("ArchaicFix", "org.embeddedt.archaicfix.ArchaicCore", "archaicfix")
```
unless there's some secret refactored ArchaicFix version I'm not aware of.

Reply:
Thanks copilot (and me not checking too closely 😓 )